crash-handler-process/main.cpp

```diff
@@ -282,6 +286,16 @@ void checkProcesses(std::mutex* m) {
 				terminalCriticalProcesses();
 
 				log_debug << "checkProcesses critical process ended" << std::endl;
+			} else if (obs_server_crash_id == 0x00001) {
+				int code = MessageBox(
+					NULL,
+					L"Out of memory."
+					L"\n\nProgram cannot allocate memory to continue function properly and will be stopped."
+					L"\nYou may considering closing some programs or reboot computer before starting a program again."
+					L"\nFind more information at howto.streamlabs.com",
```

Comment:
Suggestion for the message wording:

`The application can't allocate memory to continue functioning properly.`
`Consider closing other programs that may be consuming a lot of resources before starting the application again.`